apps/src/music/player/ToneJSPlayer.ts

```diff
@@ -375,8 +375,10 @@ class ToneJSPlayer implements AudioPlayer {
   ): BarsBeatsSixteenths {
     const bar = Math.floor(playbackPosition);
     const beat = Math.floor((playbackPosition - bar) * 4);
-    const sixteenths = Math.floor((playbackPosition - bar - beat / 4) * 16);
```

Comment:
Was this failing sometimes because the `floor` was scheduling the sound too far in the past?

Reply:
Yep I believe so, and it wasn't immediately clear because the sound would still appear on the timeline at the precise position